### ISSUE TITLE
How do I increase the size of the DAG file

### ISSUE THREAD

**Issue by GuanLawrence** [yuriy0803/open-etc-pool-friends#69]
hi all

When the epoch is zero, the initial size of the DAG file is 1024MB, and I can mine in this private pool.

What should I do if I want to increase the size of the DAG file to 3.XGB and continue mining in this private mining pool?

Thanks

**Comment by yuriy0803** [comment]
It is calculated by block height

**Comment by GuanLawrence** [comment]
hi yuriy0803

If I synchronize the etc main network and increase the height of blocks, is it feasible??

**Comment by yuriy0803** [comment]
No, if you change something in the pool, you won't get anything  dag size must be the same as geth and pool otherwise you will get unfavorable shares
